xml/ha_concepts.xml

```diff
@@ -160,6 +160,36 @@
     virtual servers and services running in physical servers. Guest
     systems can be managed as services by the cluster.
    </para>
+   <important>
+    <title>Live migration in &ha; clusters</title>
+    <para>
+     Use caution when performing live migration in an active cluster. The cluster stack
```

Comment:
It occurred to me, to clearly differentiate from the case where VMs are "managed by" cluster, this is about VMs being cluster nodes. 

We should probably somehow be more clear about this for example "live migration of cluster nodes in an active cluster" ?